hist/hist/src/THStack.cxx

```diff
@@ -555,7 +555,7 @@ Double_t THStack::GetMinimum(Option_t *option, Double_t minval)
 
    if (!opt.Contains("nostack")) {
       BuildStack();
-      h = (TH1*)fStack->At(nhists-1);
+      h = (TH1*)fStack->At(0);
```

Comment:
This fixes only normal usecase - where **ALL** histograms have not-negative bins content.

If one want to support histograms with negative bins - all of them should be checked for minimum.
Same is true for maximum value

Reply:
(TH1*)fStack->At(0) contains the sum of all histograms in the stack (computed with TH1::Add) so the minimum of this histogram is the real minimum of all the histograms.

Reply:
>  so the minimum of this histogram is the real minimum of all the histograms.

No. 

Example of three histograms. And first bin of each histogram has:

1st histogram: 1
2nd histogram: -2
3rd histogram: 3

Global minimal value of all stacked histograms will be in 2nd histogram

Reply:
Yes my fix is not right.

Reply:
Indeed see my comment here: https://root-forum.cern.ch/t/thstack-th2-with-negative-values/16161/5

Native content are not supported ... It should be documented.

Reply:
I am making a PR in that sense...